app/assets/javascripts/admin/addon/routes/admin-route-map.js

```diff
@@ -130,6 +130,7 @@ export default function () {
         "adminWebHooks",
         { path: "/web_hooks", resetNamespace: true },
         function () {
+          this.route("new");
```

Comment:
In the old implementation, new and edit routes were mashed together with a magic string "new" being passed to the edit route.